pcicrawler/cli.py

```diff
@@ -177,6 +463,13 @@ def is_physfn(device):
 
 @click.command()  # noqa: C901
 @click.option("--json/--no-json", "-j", default=False, help="Output in JSON format")
+@click.option(
+    "--ocp",
+    "-o",
```

Comment:
nit; kinda confusing, `-o` is usually short for `output`. maybe just keep the long form here if you dont want to refactor to an `-o json`/`-o ocp` arg

Reply:
taking out the -o option